q2_types/feature_table/_type.py

```diff
@@ -30,5 +30,10 @@
 
 plugin.register_semantic_type_to_format(
```

Comment:
Yeah I realized my first comment wasn't clear - this registration is for the `default` directory format for a semantic type, so we only support one default per semantic type, and we want the default to be v2.1. We would still be able to read and write v1 files, they just wouldn't be the default used for the semantic type.